src/java.desktop/aix/native/libawt/porting_aix.c

```diff
@@ -46,7 +46,7 @@ static int dladdr_dont_reload(void* addr, Dl_info* info) {
   memset((void *)info, 0, sizeof(Dl_info));
   for (;;) {
     if (addr >= p->ldinfo_textorg &&
-        addr < p->ldinfo_textorg + p->ldinfo_textsize) {
+        (char*)addr < (char*)(p->ldinfo_textorg) + p->ldinfo_textsize) {
```

Comment:
What is being warned about here?  At worst, could you just cast the RHS to `void*`?

Reply:
/porting_aix.c:49:34: error: arithmetic on a pointer to void is a GNU extension [-Werror,-Wgnu-pointer-arith]
        addr < p->ldinfo_textorg + p->ldinfo_textsize) {

and with` void*` cast on RHS
porting_aix.c:49:43: error: arithmetic on a pointer to void is a GNU extension [-Werror,-Wgnu-pointer-arith]
        addr < (void*)(p->ldinfo_textorg) + p->ldinfo_textsize) {

So either my code change or disabling warning gnu-pointer-arith. What would you prefer? and if you prefer disabling the Warning, where should I do it?